benchmarks/Smalltalk/run.st

```diff
@@ -1,5 +1,5 @@
 | args |
 "Adapt array to expectation of harness"
-args := Array with: nil withAll: Smalltalk arguments.
+args := {nil}, Smalltalk arguments.
```

Comment:
with:withAll: doesn't exist in Squeak, but the literal array syntax also works in Pharo